fixtures/external-service-bindings-app/package.json

```diff
@@ -0,0 +1,41 @@
+{
+	"name": "external-service-bindings-app",
+	"private": true,
+	"description": "A test for external service bindings",
+	"scripts": {
+		"mod-a-dev": "wrangler dev module-worker-a/index.ts --local --port 8500",
+		"mod-b-dev": "wrangler dev module-worker-b/index.ts --local --port 8501",
+		"ser-a-dev": "wrangler dev service-worker-a/index.ts --local --port 8502",
+		"mod-c-dev": "wrangler dev module-worker-c/index.ts --local --port 8503 --env staging",
+		"mod-d-dev": "wrangler dev module-worker-d/index.ts --local --port 8504 --env production",
+		"pages-dev": "cd pages-functions-app && npx wrangler pages dev public --port 8505 --service MODULE_A_SERVICE=module-worker-a MODULE_B_SERVICE=module-worker-b SERVICE_A_SERVICE=service-worker-a STAGING_MODULE_C_SERVICE=module-worker-c@staging STAGING_MODULE_D_SERVICE=module-worker-d@staging",
+		"dev": "npx concurrently -s first -k 'npm run mod-a-dev' 'npm run mod-b-dev' 'npm run ser-a-dev' 'npm run mod-c-dev' 'npm run mod-d-dev' 'npm run pages-dev'",
+		"test": "npx vitest run",
+		"test:ci": "npx vitest run",
+		"test:watch": "npx vitest",
+		"type:tests": "tsc --noEmit"
+	},
+	"jest": {
+		"restoreMocks": true,
+		"testRegex": ".*.(test|spec)\\.[jt]sx?$",
+		"testTimeout": 30000,
+		"transform": {
+			"^.+\\.c?(t|j)sx?$": [
+				"esbuild-jest",
+				{
+					"sourcemap": true
+				}
+			]
+		},
+		"transformIgnorePatterns": [
+			"node_modules/(?!find-up|locate-path|p-locate|p-limit|yocto-queue|path-exists|execa|strip-final-newline|npm-run-path|path-key|onetime|mimic-fn|human-signals|is-stream)"
+		]
+	},
```

Comment:
Is this actually used?

Reply:
no I don't think so, good catch, thanks! 😄